_posts/2024-01-22-what-makes-a-design-system-good.md

```diff
@@ -0,0 +1,53 @@
+
+HDS is quite well known and respected among open-source design systems, and an award for Best Design System in the 2023 Grand One competition serves as proof of that. Design systems are especially helpful for big organizations as they supply a unified look and user experience across all their services. The HDS team consists of a product owner, couple of designers, and a few frontend developers, some of which rotate from a developer group that maintains and develops all the projects at the city of Helsinki. I find full-time team members valuable in gaining in-depth understanding of the complexities of the design system, while the rotating members spread knowledge and usage of HDS components, all the while obtaining valuable insight on how the components work and what could be done better.
+
+In this blog post I, as a frontend developer, will give my opinion on what I consider are the most important aspects of a robust design system that offers value to its users.
+ 
+## Documentation
+ 
+Documentation is the first place the users will flock to when they are interested in or about to use a design system. Best place for docs is a dedicated website in my opinion, made with easy-to-use frameworks such as Next.js or Gatsby. First off, the home page or the navigation bar should offer links to the most important parts of the design system. To figure out what are the important parts, we need to understand the different roles in the target audience. Developers implement the components, so they need information about the components and how to get started. Designers want to look at the designs, guidelines, and patterns on how to approach different use cases such as form validation. Product owners are interested in why they should use the design system and how it would improve their product’s development on a day-to-day basis. Documentation is a benefit for the team behind the design system as well as it helps newcomers to get familiar with it. When we have figured out the needs of the users, all we need to do is provide them with quick paths to satisfaction. A good example of a documentation site is <a href="https://hds.hel.fi/" target="_blank">hds.hel.fi</a> which received positive feedback for being clear and informative. People especially liked components docs because of live code editors which allow users to test the components in a quick and easy fashion.
```

Comment:
why does the technology used to create the website *for* the DS affect if the DS is good?
do they have some unique advantage that you could elaborate on